### PR TITLE
Update tenant-roadmap-microsoft-365.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to tenant Isolation URI

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9627

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/tenant-roadmap-microsoft-365.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ To plan for your tenant implementation:
 - [Plan for client app support](microsoft-365-client-support-certificate-based-authentication.md)
 - [Determine how to use hybrid modern authentication](hybrid-modern-auth-overview.md)
 - [Plan for Office 2007 and Office 2010 upgrades](plan-upgrade-previous-versions-office.md)
-- [Understand tenant isolation](/microsoft-365-isolation-in-microsoft-365?view=o365-worldwide&preserve-view=true)
+- [Understand tenant isolation](/compliance/assurance/assurance-microsoft-365-isolation-controls?view=o365-worldwide#tenant-isolation)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/tenant-roadmap-microsoft-365.md]:
```suggestion
- [Understand tenant isolation](/compliance/assurance/assurance-microsoft-365-isolation-controls#tenant-isolation)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**